mirrord/layer/src/load.rs

```diff
@@ -1,18 +1,40 @@
+use std::{collections::HashSet, sync::LazyLock};
+
 use mirrord_config::{util::VecOrSingle, LayerConfig};
 use tracing::trace;
 
+static BUILD_TOOL_PROCESSES: LazyLock<HashSet<&str>> = LazyLock::new(|| {
+    HashSet::from([
+        "cc",
+        "ld",
+        "go",
+        "air",
+        "asm",
+        "math",
+        "cargo",
+        "hpack",
+        "rustc",
+        "compile",
+        "cargo-watch",
+    ])
+});
```

Comment:
Some other tools we skip in the VS Code extension: `[dlv, debugserver, asm, cgo, link, git, gcc, as, collect2, cc1]`
Maybe it's worth it to include these as well